src/arche/rules/result.py

```diff
@@ -1,9 +1,11 @@
 from dataclasses import dataclass, field
 from enum import Enum
-from typing import Optional
+from typing import Dict, List, Optional, Union
 
 import pandas as pd
 
+Stat = Union[pd.Series, pd.DataFrame]
```

Comment:
This is a type hints, right? 
So now Stat cover both types

Reply:
Yep, at the moment it's only for readability and documentation.
It means there will be no error if `stat` contains anything but those two.